analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Operations.cs

```diff
@@ -809,19 +809,19 @@ public void TypeOf_SetsNotNull()
         }
 
         [DataTestMethod]
-        [DataRow("bool", "true", ConstraintKind.BoolFalse)]
-        [DataRow("bool", "false", ConstraintKind.BoolTrue)]
-        [DataRow("bool?", "default", ConstraintKind.ObjectNull)]
-        [DataRow("bool?", "null", ConstraintKind.ObjectNull)]
-        public void Unary_Not_SupportsBoolAndNull(string type, string defaultValue, ConstraintKind expectedConstraint)
+        [DataRow("bool", "true", new[] { ConstraintKind.ObjectNotNull, ConstraintKind.BoolFalse })]
+        [DataRow("bool", "false", new[] { ConstraintKind.ObjectNotNull, ConstraintKind.BoolTrue })]
+        [DataRow("bool?", "default", new[] { ConstraintKind.ObjectNull })]
+        [DataRow("bool?", "null", new[] { ConstraintKind.ObjectNull })]
+        public void Unary_Not_SupportsBoolAndNull(string type, string defaultValue, ConstraintKind[] expectedConstraints)
```

Comment:
Like this, we can remove the `new[] {` part above
```suggestion
        public void Unary_Not_SupportsBoolAndNull(string type, string defaultValue, params ConstraintKind[] expectedConstraints)
```